auto_nag/scripts/configs/necko_round_robin.json

```diff
@@ -47,11 +47,12 @@
     "default":
     {
         "doc": "All the dates are the duty end dates.",
-        "2019-02-21": "Michal Novotny",
-        "2019-02-28": "Junior Hsu",
-        "2019-03-07": "Valentin Gosu",
-        "2019-03-14": "Dragana Damjanovic",
         "2019-03-21": "Kershaw Chang",
         "2019-03-28": "Honza Bambas"
```

Comment:
```suggestion
        "2019-03-28": "Honza Bambas",
```

Reply:
I've been faster than you ;)
I did it in the second commit